### PR TITLE
VUE-570 typography

### PR DESCRIPTION
**Pull request by ryan-ludwig**
* Build out custom text-h1, text-h2, etc. classes which adapt at breakpoints to match Figma spec.
* Add link styling
* Add a kitchen sink file to start working against in Storybook.

### CODE REVIEW

File: kv-tokens/configs/tailwind.config.js [hunk 1]
```diff
@@ -15,9 +15,15 @@ const {
 	radii,
 } = designtokens;
 
+const remCalc = (px) => `${px / 16}rem`;
+
 module.exports = {
 	purge: [],
 	darkMode: false, // or 'media' or 'class'
+	corePlugins: {
+		fontSize: false,
```
Comment (ryan-ludwig) [inline kv-tokens/configs/tailwind.config.js:24]: This disables tailwind from generating it's default .font-sm, .font-lg, etc. classes.

Comment (mcstover) [reply kv-tokens/configs/tailwind.config.js:24]: I suspect there will be some follow on work to ensure proper differentiation across the kitchen sink of elements...

**Review by mcstover — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
:+1: great way to do it!

**Review by BoulderBrains — APPROVED** [review]
👏

**Review by eddieferrer — APPROVED** [review]
Looks good!

**Pull request merged**